scripts/start-deployPaper

```diff
@@ -10,7 +10,15 @@ handleDebugMode
 : "${PAPER_CHANNEL:=default}"
 : "${PAPER_DOWNLOAD_URL:=}"
 : "${PAPER_CUSTOM_JAR:=}"
-: "${PAPER_CONFIG_DEFAULTS_REPO:=https://raw.githubusercontent.com/dayyeeet/minecraft-default-configs/main}"
+
+# Default repository URL
+PAPER_CONFIG_DEFAULTS_REPO="https://raw.githubusercontent.com/dayyeeet/minecraft-default-configs/main"
+
+# Check if PAPER_CONFIG_REPO is set and not empty
+if [ -n "$PAPER_CONFIG_REPO" ]; then
+  log "Download config from ${PAPER_CONFIG_REPO}"
+  PAPER_CONFIG_DEFAULTS_REPO="$PAPER_CONFIG_REPO"
+fi
```

Comment:
Similar to what I have done elsewhere, have it leverage nested, conditional variable initialization like the following. Basically, it lets the shell what you're doing with the explicit conditionals.

```suggestion
: "${PAPER_CONFIG_DEFAULTS_REPO:=${PAPER_CONFIG_REPO:=https://raw.githubusercontent.com/dayyeeet/minecraft-default-configs/main}}"
```

Reply:
I tested this change, and it works 👍 thanks for the suggestion.